### PR TITLE
Clean drivetrain

### PR DESCRIPTION
**Pull request by Bennables**
it's clean now, instead of 3 methods, just an if statement

### CODE REVIEW

**Review by Chvoong24 — APPROVED** [review]
works

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 1]
```diff
@@ -61,6 +63,14 @@ public DrivetrainSubsystem(String robotName) {
     ShuffleboardLayout backLeftLayout = null;
     ShuffleboardLayout backRightLayout = null;
 
+    if (robotName.equals("hana")) {
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:66]: Nice!

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 2]
```diff
@@ -82,17 +92,8 @@ public DrivetrainSubsystem(String robotName) {
...
-          swerveModuleSpring(frontLeftLayout, frontRightLayout, backLeftLayout, backRightLayout);
-    } else if (robotName.equals("hana")) {
-      swerveModules =
-          swerveModuleHana(frontLeftLayout, frontRightLayout, backLeftLayout, backRightLayout);
-    } else {
-      swerveModules =
-          swerveModuleCalliope(frontLeftLayout, frontRightLayout, backLeftLayout, backRightLayout);
-    }
-
+    swerveModules =
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:95]: Reducing code duplication is a good approach

File: src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java [hunk 3]
```diff
@@ -48,6 +48,8 @@ public class DrivetrainSubsystem extends SubsystemBase {
   // private final AHRS navx = new AHRS(SPI.Port.kMXP, (byte) 200); // NavX connected over MXP
   private final WrappedGyro gyro = new WrappedGyro(GyroType.PIGEON);
```
Comment (RobertTheBlair) [inline src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java:49]: Consider adding GyroType to the DriveTrainConstants, with the value that is correct for Hana / springtrap / calliope / new, and create the gyro in the DriveTrain constructor

**Review by RobertTheBlair — COMMENTED** [review]
looks fine

**Pull request merged**